src/privileged/notificationBar/schema.json

```diff
@@ -0,0 +1,34 @@
+[
+  {
+    "namespace": "notificationBar",
+    "description":
+      "A part of the study add-on that needs to be chrome-privileged",
```

Comment:
Can we just omit this description? I don't think it's mandatory.

Reply:
That's leftover from boilerplate, I'll remove it